### PR TITLE
Game class

### PR DESCRIPTION
**Pull request by huseyinbiyik**
Hello @bamsi ,

For this Pull Request, I added these features:

- Add Game Class
- Setup the constructor of it

Thank you for your review in advance.

### CODE REVIEW

File: game.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+require 'item'
+class Game < Item
+  def initialize(multiplayer, last_date_at, *attributes)
+    @multiplayer = multiplayer
+    @last_date_at = last_date_at
```
Comment (bamsi) [inline game.rb]: this should be last_played_at instead of last_date_at as it appears in the UML

**Review by bamsi — CHANGES_REQUESTED** [review]
Hello,
Good job so far, the task is almost complete, however there is a slight change you need to update as shown in the comment

Happy coding!

File: game.rb (hunk 1 above)
Comment (huseyinbiyik) [reply game.rb]: Thank you for your attention @bamsi . Fixed it now.

**Review by bamsi — APPROVED** [review]
Hello, good job so far, you can merge it

**Pull request merged**